code/build/template/[+python pyproject.toml]

```diff
@@ -0,0 +1,2 @@
+[tool.pyright]

```

Comment:
This is used by `pyright`, a type checker for Python. There are no `*.py` sources for the core engine bindings so the type checker will warn or error on some settings otherwise.

Reply:
Can't this be initialized programmatically instead?

Reply:
I can remove the configuration file. It's something that was more necessary when I started the extension, but the tooling has advanced since then and people may want to use other tooling for their own projects.